app/views/course/achievements/_form.html.slim

```diff
@@ -7,4 +7,30 @@
 
   = f.input :published
 
+  - if @achievement && !@achievement.new_record?
+    h2
+      span = t('course.condition.header')
+      div.dropdown.pull-right
+        button.btn.btn-primary.dropdown-toggle#dropdownBtn [data-toggle="dropdown"
```

Comment:
You can skip the square brackets. If you want to indicate a continuation you can use a trailing \